testsuite/test-runner/src/main/java/org/wildfly/core/testrunner/Server.java

```diff
@@ -66,8 +66,8 @@
     private final String managementProtocol = System.getProperty("management.protocol", "remote+http");
 
     // timeouts
-    private final int startupTimeout = Integer.getInteger("server.startup.timeout", 30);
-    private final int stopTimeout = Integer.getInteger("server.stop.timeout", 10);
+    private final int startupTimeout = Integer.getInteger("server.startup.timeout", 60);
```

Comment:
I'm fine with doubling these values.

I also wonder whether we could introduce a mechanism similar to `org.jboss.as.test.shared.TimeoutUtil` to influence the timeouts with the `ts.timeout.factor` system property?

Reply:
Makes sense. We'd need to port TimeoutUtil though as I don't think we want to add the dep on testsuite-shared.  That's no big deal though.

Reply:
I updated the commit with the relevant code added directly to the Server class.

Reply:
I'm pretty sure the `TimeoutUtil` is already in core right? https://github.com/wildfly/wildfly-core/blob/master/testsuite/shared/src/main/java/org/jboss/as/test/shared/TimeoutUtil.java

Reply:
That's correct, it's in core but for classes shared by the test suite. afaict, the testrunner does not depend on these shared classes.